common/src/main/java/com/wynntils/models/lootrun/type/LootrunLocation.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.lootrun.type;
+
+import com.wynntils.utils.type.BoundingBox;
+import net.minecraft.core.Position;
+
+public enum LootrunLocation {
+    SILENT_EXPANSE(400, -1150, 1600, -180),
+    CORKUS(-2150, -3500, -1200, -2100);
```

Comment:
Not the prettiest way to get this location, but this is the best we can do to get this reliably.

Reply:
Do we need these massive locations? The starting NPC only has a range of like 4-5 blocks radius

Reply:
Yes, since you can class and just rejoin anywhere. These coords include the full regions.